cmd/yggctl/actions.go

```diff
@@ -208,14 +211,23 @@ func workersAction(c *cli.Context) error {
 		fmt.Println(string(data))
 	case "table":
 		writer := tabwriter.NewWriter(os.Stdout, 4, 4, 2, ' ', 0)
-		fmt.Fprintf(writer, "WORKER\tFEATURES\n")
+		_, err = fmt.Fprint(writer, "WORKER\tFEATURES\n")
+		if err != nil {
+			return cli.Exit(fmt.Errorf("cannot write table with errors: %w", err), 1)
+		}
 		for worker, features := range workers {
 			featureSummary, err := json.Marshal(features)
 			if err != nil {
 				return cli.Exit(fmt.Errorf("cannot marshal features: %v", err), 1)
 			}
-			fmt.Fprintf(writer, "%v\t%v\n", worker, string(featureSummary))
-			_ = writer.Flush()
+			_, err = fmt.Fprintf(writer, "%v\t%v\n", worker, string(featureSummary))
```

Comment:
```suggestion
			_, err := fmt.Fprintf(writer, "%v\t%v\n", worker, string(featureSummary))
```

Reply:
No, it is not necessary.

Reply:
Actually, no change in `=` and `:=` in case of `err` is. It’s only a matter of consistency.